src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -2373,7 +2373,11 @@ private void getEEInfo(ref CORINFO_EE_INFO pEEInfoOut)
             pEEInfoOut.inlinedCallFrameInfo.size = this.SizeOfPInvokeTransitionFrame;
 
             pEEInfoOut.offsetOfDelegateInstance = (uint)pointerSize;            // Delegate::m_firstParameter
+#if READYTORUN
```

Comment:
Might be nice to pick a single scheme and stick with it.

TargetABI (15 lines below) is set by defining a property in the partial files. This is under an IFDEF. Can we have a consistent pattern?

Reply:
Fixed in 3rd commit, thanks for the suggestion.